scripts/publish-cmds.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -e
+
+if [[ -z "$1" ]]; then
+  echo -e "version is not set, run:\n"
+  echo "$0 [version] [prod|dev]"
+  exit 1
+fi
+
+if [[ -z "$2" ]]; then
+  echo -e "release type is not set, run:\n"
+  echo "$0 [version] [prod|dev]"
+  exit 1
+fi
+
+
+LDFLAGS="-X 'github.com/mesg-foundation/engine/version.Version=$1'"
+archs=(amd64 386)
+oss=(linux darwin)
+
+for os in ${oss[*]}; do
+  for arch in ${archs[*]}; do
+    echo "Building $os $arch..."
+    GOOS=$os GOARCH=$arch go build -mod=readonly -o "./bin/mesg-daemon-$os-$arch" -ldflags="$LDFLAGS" ./cmd/mesg-daemon/
+    GOOS=$os GOARCH=$arch go build -mod=readonly -o "./bin/mesg-cli-$os-$arch" -ldflags="$LDFLAGS" ./cmd/mesg-cli/
```

Comment:
let's disabled CGO to be sure the cross-compile works:
```suggestion
    CGO_ENABLED=0 GOOS=$os GOARCH=$arch go build -mod=readonly -o "./bin/mesg-daemon-$os-$arch" -ldflags="$LDFLAGS" ./cmd/mesg-daemon/
    CGO_ENABLED=0 GOOS=$os GOARCH=$arch go build -mod=readonly -o "./bin/mesg-cli-$os-$arch" -ldflags="$LDFLAGS" ./cmd/mesg-cli/
```